lib/ads/interstitial_ad_manager.js

```diff
@@ -971,10 +971,11 @@ shaka.ads.InterstitialAdManager = class {
             }
           }
         }
-        for (const asset of dataAsJson['ASSETS']) {
+        for (let i = 0; i < dataAsJson['ASSETS'].length; i++) {
+          const asset = dataAsJson['ASSETS'][i];
           if (asset['URI']) {
             interstitialsAd.push({
-              id,
+              id: id + '_asset_' + i,
```

Comment:
Theoretically, couldn't this still result in id collisions? If (for example) they have an asset list interstitial with the id "foo" and an asset uri interstitial with the id "foo_asset_0".
That seems extremely unlikely, however, so this is probably fine.